README.md

```diff
@@ -1 +1,177 @@
- # cvs-svc-document-retrieval
+# cvs-svc-document-retrieval
+
+Lambda to retrieve test certificates for commercial vehicles
+
+## Dependencies
+
+The project runs on node 12.x with typescript and serverless framework. For further details about project dependencies, please refer to the `package.json` file.
```

Comment:
We can double check with DevOps regarding the node version as I believe it is node 10 for other services